src/frontend/src/app/components/create-thread/create-thread.component.ts

```diff
@@ -47,7 +47,7 @@ export class CreateThreadComponent implements OnInit {
 
       this.newThread = {
         id: this.displayedSection.threads.length,
-        userId: this.userService.activeUser.id,
+        userId: 0,
```

Comment:
Default ID, or why zero?

Reply:
That's mock data, it will be removed when we connect the backend
The change was made because there's no activeUser field anymore